lib/utils.js

```diff
@@ -15,13 +15,42 @@ module.exports.extractKVBindings = (setFlagValues) => {
     const key = kvComponents[0]
     const value = kvComponents.slice(1).join('=')
 
+    let filepath
+
+    if (setFlagPaths) {
+      // Find a variable path from the paths which matches the variable
+      const variablePath = setFlagPaths.find(value => {
```

Comment:
Let's build an object of variables to paths outside of this for loop. We can do all the format checking there rather than in two places (here and below). Then all that's needed here is a lookup into that object.

Reply:
Totally agree - was rushed earlier